portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -52,4 +75,20 @@ private String convertToJsonUsingGson(List<String> comments) {
     String json = gson.toJson(comments);
     return json;
   }
+
+  /**
+   * Checks if the user's comment is not empty or the placeholder text
+   */
+  public boolean isValid(String comment) {
+      if (comment.isEmpty()) {
+          return false;
+      }
+      if (comment.isBlank()) {
+          return false;
+      }
+      if (comment.equalsIgnoreCase("New Comment")) {
+          return false;
+      }
+      return true;  
```

Comment:
minor - you could also just do something succinct:
`return !comment.isEmpty() && !comment.isBlank() && !comment.equalsIgnoreCase("New Comment")`

Reply:
True!